### PR TITLE
Remove unnecessary trusted lambdas

### PR DESCRIPTION
**Pull request by n8sh**
If everything a function does is wrapped in a trusted lambda instead mark the function as trusted.

### CODE REVIEW

File: src/object.d [hunk 1]
```diff
@@ -842,14 +842,14 @@ class TypeInfo_Vector : TypeInfo
 
 class TypeInfo_Function : TypeInfo
 {
-    override string toString() const
+    override string toString() const @trusted
```
Comment (Geod24) [inline src/object.d:845]: Doesn't that change the API ? Meaning all overrides need to be `@safe` (or `@trusted`) ?

Comment (n8sh) [reply src/object.d:845]: It doesn't because `TypeInfo.toString` is already `@safe`.

**Review by thewilsonator — APPROVED** [review]

**Comment by 12345swordy** [comment]
This hasn't been merge for some reason.

**Review by Geod24 — APPROVED** [review]

**Comment by thewilsonator** [comment]
@Geod24 There seems to be a lack of communication between the autotester and GH. This is all green but still not merging. It will likely have to be force merged.

**Comment by Geod24** [comment]
Auto-merge toggled on

**Comment by Geod24** [comment]
Well, it was worth a try.

**Pull request merged**